### PR TITLE
[PRM-573] Added CalculationView with table

### PR DESCRIPTION
**Pull request by JBlacklaws**

### CODE REVIEW

**Comment by JBlacklaws** [comment]
image

**Review by BackRowOla — APPROVED** [review]
Looks good, just needs an IT test for the new controller and a small change comment.

Also discussed the content for calculation.key3 on slack. Waiting on response in requirements channel.

**Comment by JBlacklaws** [comment]
image

File: it/controllers/CalculationControllerISpec.scala [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+import play.api.test.FakeRequest
+import play.api.test.Helpers._
+import stubs.AuthStub
+import testUtils.IntegrationSpecCommonBase
+import utils.SessionKeys
+
+class CalculationControllerISpec extends IntegrationSpecCommonBase {
+
+  val controller = injector.instanceOf[CalculationController]
+  val fakeAgentRequest = FakeRequest("GET", "/calculation").withSession(SessionKeys.agentSessionVrn -> "123456789")
```
Comment (BackRowOla) [inline it/controllers/CalculationControllerISpec.scala:30]: Controller and agent request val not used, is it needed?

**Review by BackRowOla — APPROVED** [review]

**Pull request merged**